src/ui/settings/ViewSubscriptionManagement/ViewSubscriptionManagement.qml

```diff
@@ -232,29 +233,28 @@ VPNFlickable {
         );
     }
 
-    function getPlanText(currencyCode, amount, intervalCount) {
+    function getPlanText(currencyCode, amount) {
         const amountDisplay = (amount || 0) / 100;
         const localizedCurrency = VPNLocalizer.localizeCurrency(amountDisplay, currencyCode);
 
-        let labelText;
-
-        if (intervalCount === 12) {
-            // {¤amount} Yearly
-            labelText = VPNl18n.SubscriptionManagementPlanValueYearly.arg(localizedCurrency);
-        } else if (intervalCount === 6) {
-            // {¤amount} Half-yearly
-            labelText = VPNl18n.SubscriptionManagementPlanValueHalfYearly.arg(localizedCurrency);
-        } else if (intervalCount === 1) {
-            // {¤amount} Monthly
-            labelText = VPNl18n.SubscriptionManagementPlanValueMonthly.arg(localizedCurrency);
-        } else {
-            console.warn(`Unexpected value for intervalCount: ${intervalCount}`);
-            VPN.recordGleanEventWithExtraKeys("unhandledSubPlanInterval", {
-                "interval_count": intervalCount,
-            });
+        switch (VPNSubscriptionData.planBillingInterval) {
+            case VPNSubscriptionData.BillingIntervalMonthly:
+                // {¤amount} Monthly
```

Comment:
what is that symbol after '{'?

Reply:
It’s a [currency sign](https://en.wikipedia.org/wiki/Currency_sign_(typography)) that can be used as a placeholder for example.